common/walker/src/report.rs

```diff
@@ -126,6 +129,86 @@ impl SplitScannerError for Result<Report, ScannerError> {
 /// Handle the report
 pub async fn handle_report(report: Report) -> anyhow::Result<()> {
     // FIXME: this is a very simplistic version of handling the error
-    log::info!("Import report: {report:#?}");
+    log::warn!("Import report: {report:#?}");
```

Comment:
We should not log the whole report to the standard output anymore. Better to log the location of the report when successfully created.